Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -374,6 +374,45 @@ namespace O3DE::ProjectManager
         }
     }
 
+
+    void GemItemDelegate::DrawPlatformText(QPainter* painter, const QRect& contentRect, const QFont& standardFont, [[maybe_unused]] const QModelIndex& modelIndex) const
```

Comment:
you are using this param so you don't need `maybe_unused`
```suggestion
    void GemItemDelegate::DrawPlatformText(QPainter* painter, const QRect& contentRect, const QFont& standardFont, const QModelIndex& modelIndex) const
```